Cargo.toml

```diff
@@ -24,6 +24,7 @@ digest = "0.9"
 jubjub = "0.6"
 rand_core = "0.6"
 serde = { version = "1", optional = true, features = ["derive"] }
+serde_json = "1.0"
```

Comment:
Most users will want each FROST message as a single byte array, so they can embed them in another protocol.

So it's ok to use `serde_json` in the tests (`[dev-dependencies]` section). But we shouldn't require it for every user of this crate.